runtime/core/websocket/websocket_server.h

```diff
@@ -60,14 +59,15 @@ class ConnectionHandler {
   void operator()();
 
  private:
-  int nbest_;
+  int nbest_ = 1;
   websocket::stream<tcp::socket> ws_;
   std::shared_ptr<FeaturePipelineConfig> feature_config_;
   std::shared_ptr<DecodeOptions> decode_config_;
   std::shared_ptr<SymbolTable> symbol_table_;
   std::shared_ptr<TorchAsrModel> model_;
 
   bool got_start_tag_ = false;
+  bool got_end_tag_ = false;
```

Comment:
change it to stop_recognition_
and add some comment such as
// When endpoint is detected, we should stop recognition and stop receiving data.